### PR TITLE
Fix issue ArabicStemmer AttributeError #1852

### PR DESCRIPTION
**Pull request by greenat92**
Hi @alvations, @richbalmer
1. Add arabic stop word list.
2. Fix issue ArabicStemmer AttributeError #1852

### CODE REVIEW

**Comment by alvations** [comment]
[CI: retest]

**Comment by stevenbird** [comment]
@LBenzahia would you please share the list of stopwords as a newline-delimited list of words, and I'll add it to the NLTK stopwords corpus, and you can access it there.

**Comment by greenat92** [comment]
@stevenbird,

>  would you please share the list of stopwords as a newline-delimited list of words

Indeed, I've added the stopwords list see this [PR]( https://github.com/nltk/nltk_data/pull/95)

**Comment by stevenbird** [comment]
@LBenzahia: Thanks, I've added the Arabic stopwords to the NLTK corpus collection.

**Comment by alvations** [comment]
[CI: retest]

**Comment by alvations** [comment]
With the new stopwords and the CI retest, the tests passed. 

@LBenzahia Could you help to take a look and is the PR set for a final review before merge?

**Comment by greenat92** [comment]
@alvations Thanks , I've tested it locally LGTM :+1:, If there's any problem let me know to fix it.

File: nltk/test/unit/test_stem.py
Comment (alvations) [inline nltk/test/unit/test_stem.py:19]: Please add another test where the `ignore_stopwords=False`.

File: nltk/stem/snowball.py
Comment (alvations) [inline nltk/stem/snowball.py:765]: In which cases would there be more steps that needs to be applied here? Perhaps, it'll be good to list these cases down.

File: nltk/stem/snowball.py
Comment (alvations) [inline nltk/stem/snowball.py:757]: 👍

File: nltk/stem/snowball.py
Comment (alvations) [inline nltk/stem/snowball.py:519]: Just out of curiosity, is there a linguistic reason to avoid words with 2 characters?

File: nltk/stem/snowball.py
Comment (greenat92) [reply nltk/stem/snowball.py:765]: We're  working on it and other todos when we solve them we'll send PR for updates 
"or next"  i mean this [line](https://github.com/assem-ch/arabicstemmer/blob/86f9046451e079da40d85106f65cb9194b055aff/algorithm/stemmer.sbl#L586) from the original algorithm as you know i've rewrote the algorithm by hand to follow nltk guideline style code and avoid the generated code from snowball generator.  
You can take a look at this list of [issues and todos](https://github.com/assem-ch/arabicstemmer/issues/1).

File: nltk/stem/snowball.py
Comment (greenat92) [reply nltk/stem/snowball.py:519]: We didn't study the case of words that have 2 length yet,  We're mentioned it in the list of our todos too.

File: nltk/test/unit/test_stem.py
Comment (LBenzahia) [reply nltk/test/unit/test_stem.py]: WIP

File: nltk/stem/snowball.py
Comment (alvations) [reply nltk/stem/snowball.py:765]: Could you add a link to the https://github.com/assem-ch/arabicstemmer/issues/1 in the github comment too? That'll be helpful for us to track later. Thanks!

**Comment by stevenbird** [comment]
@LBenzahia: I think we're waiting on more input from you before merging.

File: nltk/stem/snowball.py
Comment (greenat92) [reply nltk/stem/snowball.py:765]: I've created an [issue](https://github.com/nltk/nltk/issues/1921) in nltk to track the changes later and added a comment in [https://github.com/assem-ch/arabicstemmer/issues/1](https://github.com/assem-ch/arabicstemmer/issues/1) , I hope this is helpful, sorry for the late replay.

**Comment by greenat92** [comment]
@stevenbird, I've created an [issue](https://github.com/nltk/nltk/issues/1921) for that and linked it with our milestone for improving snowball ArabicStemmer in the original repo of the stemmer, Sorry for the late replay.
Thank you!

**Review by alvations — APPROVED** [review]

**Comment by alvations** [comment]
[CI: retest]

**Comment by alvations** [comment]
[CI: retest]

**Comment by assem-ch** [comment]
What's still in this pr to be merged?

**Comment by alvations** [comment]
[CI: retest]

**Comment by alvations** [comment]
@stevenbird @assem-ch I think it LGTM if no one else objects.

**Comment by stevenbird** [comment]
Thanks @LBenzahia

**Pull request merged**
